lib/ui/platform_dispatcher.dart

```diff
@@ -1479,7 +1479,7 @@ class _ViewConfiguration {
   /// The pixel density of the output surface.
   final double devicePixelRatio;
 
-  /// The size requested for the view in logical pixels.
+  /// The size requested for the view in physical pixels.
```

Comment:
How did we find out this should be physical instead of logical? Is it a bug fix or a behavior change?

Reply:
The value was always in physical pixels, it was just documented wrong. Luckily, it is a doc comment on a private class, so the doc was never actually public.

Reply:
Backup evidence that this has always been in physical pixels and we just documented it wornt: This value is returned by the physicalSize getter on `FlutterView`:

https://github.com/flutter/engine/blob/dba80be699446ee6142038a9d613e367f53ffa7d/lib/ui/window.dart#L163

Reply:
I've found Physical vs Logical pixels is *painful* in the web, especially if one trusts the docs :P